pkg/chains/rekor.go

```diff
@@ -68,8 +68,10 @@ func publicKeyOrCert(signer signing.Signer, cert string) ([]byte, error) {
 	return pem, nil
 }
 
-// for testing
-var getRekor = func(url string, l *zap.SugaredLogger) (rekorClient, error) {
+var getRekor = func(is_enabled bool, url string, l *zap.SugaredLogger) (rekorClient, error) {
```

Comment:
Would it be more straight forward to only call `getRekor` if it is enabled since the caller already has that information?

Reply:
Cool! I updated the PR and moved the client initialization to be gated inside the shouldUploadTlog function. Let me know if this looks good to you. Thanks!